### PR TITLE
py-nibabel

### PR DESCRIPTION
**Pull request by eborisch**
[no]maintainer update.

### CODE REVIEW

File: python/py-nibabel/Portfile [hunk 1]
```diff
@@ -4,36 +4,51 @@ PortSystem              1.0
...
     depends_build-append \
                         port:py${python.version}-setuptools
 
     depends_lib-append  port:py${python.version}-numpy \
+                        port:py${python.version}-bz2file \
                         port:py${python.version}-six
 
+    test.run    yes
+    test.cmd    ${python.bin} setup.py
+    test.target test
```
Comment (reneeotten) [inline python/py-nibabel/Portfile]: this is the default ```test.target``` so you don't really need to specify it

File: python/py-nibabel/Portfile [hunk 2]
```diff
@@ -4,36 +4,51 @@ PortSystem              1.0
...
+    sha256  dd0c41715d0391c724e2828bba2c16690dbd6aafbca8e920ee8448ed0086e4c1 \
+    size    4589348
 
 python.versions         27 34 35 36 37
 
 if {${name} ne ${subport}} {
+    depends_test-append     port:py${python.version}-nose \
+                            port:py${python.version}-mock \
+                            port:py${python.version}-Sphinx \
+                            port:py${python.version}-pytest \
```
Comment (reneeotten) [inline python/py-nibabel/Portfile]: why did you add ```pytest```; it appears to be only using ```nose``` or?

File: python/py-nibabel/Portfile [hunk 3]
```diff
@@ -4,36 +4,51 @@ PortSystem              1.0
...
+    rmd160  a479afa81c5b79752ef0f39dcde0304e53a26f3a \
+    sha256  dd0c41715d0391c724e2828bba2c16690dbd6aafbca8e920ee8448ed0086e4c1 \
+    size    4589348
 
 python.versions         27 34 35 36 37
 
 if {${name} ne ${subport}} {
+    depends_test-append     port:py${python.version}-nose \
+                            port:py${python.version}-mock \
+                            port:py${python.version}-Sphinx \
```
Comment (reneeotten) [inline python/py-nibabel/Portfile]: this should probably be ```py-sphinx```, not with a capital? Is this really a test-dependency, it's typically only needed for building the documentation?

File: python/py-nibabel/Portfile [hunk 4]
```diff
@@ -4,36 +4,51 @@ PortSystem              1.0
 PortGroup               python 1.0
 
 name                    py-nibabel
-version                 2.3.3
+version                 2.4.0
 revision                0
 platforms               darwin
 license                 MIT
```
Comment (reneeotten) [inline python/py-nibabel/Portfile]: it appears that nothing gets compiled / architecture-dependent files are installed. If that observation is correct, then it would be good to add ```supported_archs     noarch```

File: python/py-nibabel/Portfile [hunk 5]
```diff
@@ -4,36 +4,51 @@ PortSystem              1.0
...
+    size    4589348
 
 python.versions         27 34 35 36 37
 
 if {${name} ne ${subport}} {
+    depends_test-append     port:py${python.version}-nose \
+                            port:py${python.version}-mock \
+                            port:py${python.version}-Sphinx \
+                            port:py${python.version}-pytest \
+                            port:py${python.version}-setuptools
```
Comment (reneeotten) [inline python/py-nibabel/Portfile]: ```py-setuptools``` isn't needed as a test-dependency since it's already listed under ```depends_build```.

File: python/py-nibabel/Portfile [hunk 6]
```diff
@@ -4,36 +4,51 @@ PortSystem              1.0
...
+
     depends_build-append \
                         port:py${python.version}-setuptools
 
     depends_lib-append  port:py${python.version}-numpy \
+                        port:py${python.version}-bz2file \
                         port:py${python.version}-six
 
+    test.run    yes
```
Comment (reneeotten) [inline python/py-nibabel/Portfile]: it actually doesn't pass all tests with Python 3.7, not sure if that's a "real" problem or not...

File: python/py-nibabel/Portfile [hunk 7]
```diff
@@ -4,36 +4,51 @@ PortSystem              1.0
...
 
 homepage                https://nipy.org/nibabel
 master_sites            pypi:n/nibabel/
 distname                nibabel-${version}
 
-checksums               rmd160  1f202d10b89872cb056f64334f262cfa02df0075 \
-                        sha256  b6366634c65b04464e62f3a9a8df1faa172f780ed7f1af1c6818b3dc2f1202c3 \
-                        size    4492243
+checksums \
+    rmd160  a479afa81c5b79752ef0f39dcde0304e53a26f3a \
```
Comment (reneeotten) [inline python/py-nibabel/Portfile]: I would consider keeping your whitespaces consistent throughout the portfile

File: python/py-nibabel/Portfile (hunk 3 above)
Comment (mf2k) [reply python/py-nibabel/Portfile]: Yes, it should be lower case "s" in py-sphinx to match the Portfile name case.

**Comment by eborisch** [comment]
Clearly I shouldn’t try to submit a new port while I’m at a conference. ;)

Thanks for all of your productive comments; they will be addressed.

File: python/py-nibabel/Portfile (hunk 7 above)
Comment (eborisch) [reply python/py-nibabel/Portfile]: I like to keep it under 80 columns. Personal preference. :)
